framework/src/problems/FEProblemBase.C

```diff
@@ -472,6 +472,19 @@ FEProblemBase::addExtraVectors()
   _aux->addExtraVectors();
 }
 
+ConstElemRange *
+FEProblemBase::getEvaluableElementRange()
+{
+  if (!_evaluable_local_elem_range)
+  {
+    _evaluable_local_elem_range =
+        libmesh_make_unique<ConstElemRange>(_mesh.getMesh().evaluable_elements_begin(_nl->dofMap()),
+                                            _mesh.getMesh().evaluable_elements_end(_nl->dofMap()));
+  }
+
+  return _evaluable_local_elem_range.get();
```

Comment:
Minor comment. There's no reason to return a pointer here if it's never null. I'm currently adding this to my in-progress PR but returning a reference instead.